packages/@ourworldindata/grapher/src/axis/AxisViews.tsx

```diff
@@ -83,31 +83,69 @@ export class HorizontalAxisGridLines extends React.Component<{
 
         return (
             <g className={classNames("AxisGridLines", "verticalLines")}>
-                {axis.getTickValues().map((t, i) => {
-                    const color = t.faint
-                        ? FAINT_TICK_COLOR
-                        : t.solid
-                        ? SOLID_TICK_COLOR
-                        : TICK_COLOR
+                {axis
+                    .getTickValues()
+                    .filter((t) => t.value !== 0)
+                    .map((t, i) => {
+                        const color = t.faint
+                            ? FAINT_TICK_COLOR
+                            : t.solid
+                            ? SOLID_TICK_COLOR
+                            : TICK_COLOR
 
-                    return (
-                        <line
-                            key={i}
-                            x1={axis.place(t.value)}
-                            y1={bounds.bottom.toFixed(2)}
-                            x2={axis.place(t.value)}
-                            y2={bounds.top.toFixed(2)}
-                            stroke={color}
-                            strokeDasharray={
-                                t.solid
-                                    ? undefined
-                                    : dasharrayFromFontSize(
-                                          horizontalAxis.tickFontSize
-                                      )
-                            }
-                        />
-                    )
-                })}
+                        return (
+                            <line
+                                key={i}
+                                x1={axis.place(t.value)}
+                                y1={bounds.bottom.toFixed(2)}
+                                x2={axis.place(t.value)}
+                                y2={bounds.top.toFixed(2)}
+                                stroke={color}
+                                strokeDasharray={
+                                    t.solid
+                                        ? undefined
+                                        : dasharrayFromFontSize(
+                                              horizontalAxis.tickFontSize
+                                          )
+                                }
+                            />
+                        )
+                    })}
+            </g>
+        )
+    }
+}
+
+@observer
+export class HorizontalAxisZeroLine extends React.Component<{
+    horizontalAxis: HorizontalAxis
+    bounds?: Bounds
```

Comment:
These should not be optional, when rendering the zero line using `DEFAULT_BOUNDS` the result will look very wrong, so better remove that before we shoot ourselves in the foot.